CHANGELOG.md

```diff
@@ -24,6 +32,8 @@
   the hub ([pybricks-micropython#250]).
 - Improved font for the digits ``0--9`` when displaying them
   with `hub.display.char(str(x))` ([pybricks-micropython#253]).
+- Added an optional 32-bit program identifier payload to the protocol commands
```

Comment:
Looks like leftover from the previous approach.

Reply:
Thanks!